docs/contributing.md

```diff
@@ -17,6 +17,16 @@ It is recommended that automated github actions are turned on in your fork to ha
 You can turn them on here:  
 ![Github actions example](./img/github-actions-example.png)
 
+* **When reviewing PRs, please leave a message what was done.**
+We don't have full test coverage, so manual testing can help.
+For code changes that could affect multiple worlds or that could have changes in unexpected code paths, manual testing
+or checking if all code paths are covered by automated tests is desired. The original author may not have been able
+to test all possibly affected worlds, or didn't know it would affect another world. In such cases, it is helpful to
+state which games or settings were rolled, if any.
+Please also tell us if you looked at code, just did functional testing, did both or neither.
+If the PR depends on other PRs to be tested, please state what you merged into what for testing.
```

Comment:
This could be reworded/reordered to be more clear:
```suggestion
If testing the PR depends on other PRs, please state what you merged into what for testing.
```